cupy/cuda/thrust.pyx

```diff
@@ -8,7 +8,9 @@ cimport cython  # NOQA
 from libcpp cimport vector
 
 from cupy.cuda cimport common
+from cupy.cuda cimport device
 from cupy.cuda cimport memory
+from cupy.cuda cimport runtime
```

Comment:
This cimport of runtime seems to be what is causing the failure when building with `--cupy-no-cuda`.